inc/savedsearch.class.php

```diff
@@ -334,8 +332,8 @@ function cleanDBonPurge() {
    /**
     * Print the saved search form
     *
-    * @param integer $ID      ID of the item
-    * @param array   $options Options:
+    * @param $ID         integer    ID of the item
```

Comment:
I follow GLPI CS rules.
If we change it, wez must change also for all comment of functions.